modules/social_features/social_event/social_event.install

```diff
@@ -1502,3 +1502,17 @@ function social_event_update_8807() {
     }
   }
 }
+
+/**
+ * Update the enrollment method text of an event on event add form.
+ */
+function social_event_update_8808() {
+  /** @var \Drupal\update_helper\Updater $updateHelper */
+  $updateHelper = \Drupal::service('update_helper.updater');
+
+  // Execute configuration update definitions with logging of success.
+  $updateHelper->executeUpdate('social_event', 'social_event_update_8808');
```

Comment:
Did you test this or am I doing something wrong? I get the following message
[warning] Unable to update configuration for field.storage.node.field_enroll_method.